### PR TITLE
Exercise 1 foxtrotzulu.sh

### PR DESCRIPTION
**Pull request by Foxtrot-Zulu**

### CODE REVIEW

File: exercises/week 1/solutions/foxtrotzulu.sh [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+# Print a message to terminal
+echo "Binary signature verification successful"
+
+# Copy the binaries to /usr/local/bin/
+sudo cp bitcoin-25.0-x86_64-linux-gnu.tar.gz /usr/local/bin/
+sudo tar -xzf bitcoin-25.0-x86_64-linux-gnu.tar.gz -C /usr/local/bin/
+
+# Add dir to path
+echo "export PATH=/usr/local/bin/bitcoin-25.0/bin:$PATH" >> ~/.bashrc && source ~/.bashrc
```
Comment (rajarshimaitra) [inline exercises/week 1/solutions/foxtrotzulu.sh:32]: If you instead copy the extracted binaries to the `usr/local/bin` folder, without creating the extra `bitcoin-25.0` folder, then you won't need to add it in your $PATH variable.

Comment (rajarshimaitra) [reply exercises/week 1/solutions/foxtrotzulu.sh:32]: This is generally a bad idea, as you should not alter the user's environment variable or any other system settings by the script.

**Review by rajarshimaitra — CHANGES_REQUESTED** [review]
Looks good, one comment.

**Comment by Foxtrot-Zulu** [comment]
Thanks for the corrections!

File: exercises/week 1/solutions/foxtrotzulu.sh (hunk 1 above)
Comment (Foxtrot-Zulu) [reply exercises/week 1/solutions/foxtrotzulu.sh:32]: Thanks Raj!

**Pull request closed without merging**